Documentation/gettingstarted/kubeproxy-free.rst

```diff
@@ -498,7 +498,8 @@ Socket LoadBalancer Bypass in Pod Namespace
 
 Cilium has built-in support for bypassing the socket-level loadbalancer and falling back
 to the tc loadbalancer at the veth interface when a custom redirection/operation relies
-on the original ClusterIP within pod namespace (e.g., Istio side-car).
+on the original ClusterIP within pod namespace (e.g., Istio side-car) or due to the Pod's
+nature the socket-level loadbalancer is ineffective (e.g., KubeVirt).
```

Comment:
How comprehensive do we want to be here? I think this also applies to kata.

Reply:
Good point! Updated (I think it's not a bad idea to be comprehensive/searchable for this particular case).